.ci/tweet-releasenotes.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+    if [[ "$(declare -p "$1")" =~ ^declare\ -x ]]; then
+       echo "Error: Define $1 environment variable"
+       exit 1
+    fi
+}
+
+checkForVariable "TWITTER_CONSUMER_KEY"
+checkForVariable "TWITTER_CONSUMER_SECRET"
+checkForVariable "TWITTER_ACCESS_TOKEN"
+checkForVariable "TWITTER_ACCESS_TOKEN_SECRET"
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+cd .ci-temp/checkstyle
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+| grep -oP '"tag_name": "\K(.*)(?=")')
```

Comment:
Use jq tool

Reply:
We might need to use git read only token.
Please recheck release.sh on how to do this.

Reply:
Changes Done